web/cobrands/peterborough/assets.js

```diff
@@ -225,9 +225,9 @@ fixmystreet.assets.add(flytipping_defaults, {
                     return;
                 }
             }
-            $("#js-roads-responsibility").removeClass("hidden");
-            $("#js-roads-responsibility .js-responsibility-message").addClass("hidden");
+            $("#js-roads-responsibility").addClass("hidden");
             $('#js-environment-message').removeClass('hidden');
+            fixmystreet.message_controller.check_for_stopper();
```

Comment:
check_for_stopper is called on category change anyway, why do we need to additionally call it here?